### ISSUE TITLE
Add forgot password link

### ISSUE THREAD

**Issue by kristenwilson** [k-int/gokb-phase1#187]
Add a forgot password link to the log in screen.

**Comment by kristenwilson** [comment]
Forgot password link is still missing from log in screen.

**Comment by kristenwilson** [comment]
The forgot password link is still missing from the log in screen. It just needs to be added below the log in fields.

![log-in-screen](https://cloud.githubusercontent.com/assets/7397776/5208092/c9320ade-757e-11e4-9c1b-cacce5d91498.png)

**Comment by kristenwilson** [comment]
The add password link is now present on the log in screen, but it doesn't seem to be working. I requested a password reset multiple times, and I never received an email allowing me to complete the actions. (I checked Spam too and it was not there.)

**Comment by ianibo** [comment]
Hey Kristen

Really sorry - not able to replicate this - used the link and got the email (It was in my spam tho). Not sure what to do next.. any ideas [steve?]

**Comment by kristenwilson** [comment]
So changed the email address on one of my accounts to my personal gmail address, and I got the email (but also in spam). I tested again using my NCSU email address, and the email just does not come through. So I think this is working, but NCSU must have some additional layer of spam filtering that's preventing it from being delivered. It's strange, though, because if I create a new account, the confirmation email does come though to my NCSU email. I can ask our IT department if they know anything about it. From GOKb's perspective though, I think this is working.
